class-tour.php

```diff
@@ -44,10 +44,28 @@ public static function enqueue_scripts() {
 		}
 		$once = true;
 
-		$tours = apply_filters( 'tour_list', array() );
-		if ( empty( $tours ) ) {
+		$_tours = apply_filters( 'tour_list', array() );
+		if ( empty( $_tours ) ) {
 			return;
 		}
+		$tours = array_filter(
+			$_tours,
+			function ( $tour ) {
+				$restrict_url = $tour[0]['tour_restrict_url'];
+				$url_pattern  = '#/wp-admin/post\.php\?post=#';
+				if ( empty( $tour[0]['tour_restrict_url'] ) || preg_match( $url_pattern, $_SERVER['REQUEST_URI'] ) ) {
+					return true;
+				}
+				if ( ! preg_match( '#^/.*$#', $restrict_url ) ) {
```

Comment:
Could you elaborate on this? It looks like you are doing a `substr( $restrict_url, 1 ) !== '/'` here, why regex?

Reply:
I was trying to check if the URL entered when creating the tour is in the format `/abc` and I agree that this can be better refactored.